### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by JanefrancessC**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -2,12 +2,18 @@ import { Route, Routes } from "react-router-dom";
...
 const App = () => (
+	<div className="App">
+		{/* <Login /> */}
 	<Routes>
-		<Route path="/" element={<Home />} />
+		<Route path="/" element={<Login />} />
 		<Route path="/about/this/site" element={<About />} />
 	</Routes>
+	</div>
 );
```
Comment (SaadiaELF) [inline client/src/App.js:15]: Could you please delete the unnecessary div and commented code
```js
<div className="App">
</div>

```

**Review by SaadiaELF — APPROVED** [review]
Good job. just remember to delete all unnecessary commented code :)

File: client/src/App.js (hunk 1 above)
Comment (JanefrancessC) [reply client/src/App.js:15]: I will definitely do that.

**Comment by JanefrancessC** [comment]
Modified the styling

**Review by mrvicthor — APPROVED** [review]

**Review by mrvicthor — APPROVED** [review]

**Pull request merged**